src/simulations/GenericPerturbation.jl

```diff
@@ -0,0 +1,32 @@
+using OrdinaryDiffEq: ODEProblem, Rodas4, init, solve!, step!, reinit!, savevalues!, u_modified!
+using Setfield
+
+"""
+```Julia
+GemericPerturbation(;node_number,fraction,tspan_fault)
+```
+# Keyword Arguments
+- `node`: number or name of the node
+- `var_new`: value of power during fault event
+- `tspan_fault`: PowerPerturbation timespan
+- `var`:  symbol of parameter that is perturbed
+"""
+
+Base.@kwdef struct GenericPerturbation <:NodePerturbation
```

Comment:
Maybe rename `GenericPerturbation` -> `ChangeParameter` or sth like that? it is abit misleading that the concrete type is called "generic" while the abstract type has a more specific name :-)

Reply:
maybe I change `NodePerturbation` to `AbstractNodePerturbation` and `GenericPerturbation` to `NodeParameterChange`?